examples/postgresql/README.md

```diff
@@ -141,8 +141,8 @@ $ kanctl create actionset --action backup --namespace kanister --blueprint postg
 actionset backup-llfb8 created
 
 $ kubectl --namespace kanister get actionsets.cr.kanister.io
-NAME           AGE
-backup-glptq   38s
+NAME           PROGRESS   LAST TRANSITION TIME   STATE
+backup-glptq   10.00      2022-12-08T18:14:09Z   complete
```

Comment:
The `PROGRESS` here should be marked as 100.00 instead of 10.00. I think @akankshakumari393 reported this issue as well. Could you please change this to `100.00`?

Reply:
changed